AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/AWSIncomingEventReconciliationQueue.swift

```diff
@@ -10,14 +10,32 @@ import AWSPluginsCore
 import Combine
 import Foundation
 
+//Used for testing:
+@available(iOS 13.0, *)
+typealias IncomingEventReconciliationQueueFactory =
+    ([Model.Type], APICategoryGraphQLBehavior, StorageEngineAdapter) -> IncomingEventReconciliationQueue
+
 @available(iOS 13.0, *)
 final class AWSIncomingEventReconciliationQueue: IncomingEventReconciliationQueue {
 
+    static let factory: IncomingEventReconciliationQueueFactory = { modelTypes, api, storageAdapter in
+        AWSIncomingEventReconciliationQueue(modelTypes: modelTypes, api: api, storageAdapter: storageAdapter)
+    }
+
+    private let reconciliationQueueTopic: PassthroughSubject<IncomingEventReconciliationQueueEvent, DataStoreError>
+    private var reconciliationQueueTopicCancellable = [String: AnyCancellable]()
```

Comment:
nit: I dislike mixing "declare and init" vs. "only declare" properties. Can we move the init of `reconciliationQueueTopicCancellable` into `init()`? Luckily, since you will declare the type here, the init should be as simple as `self. reconciliationQueueTopicCancellable = [:]`

Reply:
done! thanks!